projects/unicorn/build.sh

```diff
@@ -27,6 +27,6 @@ do
 
     $CXX $CXXFLAGS fuzz_$target.o -o $OUT/fuzz_$target ../../libunicorn.a -lFuzzingEngine
 
-    #TODO corpuses
+    # TODO corpuses
     cp fuzz_emu.options $OUT/fuzz_$target.options
```

Comment:
If you really want to enforce `-max_len=4096` limit, you would have to do it in the code. Please see https://github.com/google/oss-fuzz/blob/22f3a1481cbc8252fb340369cdb3d50f185017fa/docs/new_project_guide.md#custom-libfuzzer-options-for-clusterfuzz